tests/bsim/bluetooth/host/privacy/central/src/tester.c

```diff
@@ -125,3 +134,56 @@ void tester_procedure(void)
 
 	PASS("PASS\n");
 }
+
+void tester_procedure_peripheral_delayed_start_of_connectable_adv(void)
+{
+	backchannel_init(0);
+
+	int err;
+	struct bt_le_adv_param params =
+		BT_LE_ADV_PARAM_INIT(BT_LE_ADV_OPT_CONNECTABLE | BT_LE_ADV_OPT_USE_IDENTITY,
+				     BT_GAP_ADV_FAST_INT_MIN_2,
+				     BT_GAP_ADV_FAST_INT_MAX_2, NULL);
+	struct bt_data ad;
+	struct bt_le_ext_adv *adv;
+
+	/* Enable bluetooth */
+	err = bt_enable(NULL);
+	if (err) {
+		FAIL("Failed to enable bluetooth (err %d\n)", err);
+	}
+
+	/* Advertiser to use a long RPA timeout */
+	err = bt_le_set_rpa_timeout(100);
+	TEST_ASSERT(!err, "Failed to set RPA timeout (err %d)", err);
+
+	err = bt_le_ext_adv_create(&params, &adv_callbacks, &adv);
+	TEST_ASSERT(!err, "Failed to create advertising set (err %d)", err);
+
+	ad.type = BT_DATA_NAME_COMPLETE;
+	ad.data_len = strlen(CONFIG_BT_DEVICE_NAME);
+	ad.data = (const uint8_t *)CONFIG_BT_DEVICE_NAME;
+
+	err = bt_le_ext_adv_set_data(adv, &ad, 1, NULL, 0);
+	TEST_ASSERT(!err, "Failed to set advertising data (err %d)", err);
+
+	err = bt_le_ext_adv_start(adv, BT_LE_EXT_ADV_START_DEFAULT);
+	TEST_ASSERT(!err, "Failed to start advertiser (err %d)", err);
```

Comment:
Yeah, I looked into that initially. It is nice to have such a library. Here I explicitly wanted to use `BT_LE_ADV_OPT_USE_IDENTITY` so that the advertiser would use the same address when restarting

Reply:
That makes sense. Maybe the testlib can/should be expanded to support more options